chips/lowrisc/src/hmac.rs

```diff
@@ -269,3 +277,15 @@ impl hil::digest::HMACSha256 for Hmac<'_> {
         Ok(())
     }
 }
+
+impl hil::digest::HMACSha384<32> for Hmac<'_> {
+    fn set_mode_hmacsha384(&self, _key: &[u8; 32]) -> Result<(), ErrorCode> {
+        Err(ErrorCode::NOSUPPORT)
+    }
+}
+
+impl hil::digest::HMACSha512<32> for Hmac<'_> {
+    fn set_mode_hmacsha512(&self, _key: &[u8; 32]) -> Result<(), ErrorCode> {
+        Err(ErrorCode::NOSUPPORT)
+    }
+}
```

Comment:
I don't love the pattern of implementing traits to just return errors, but I understand the design of the virtualizer sort of requires this (at least without adding a decent amount of complexity).